interface/main/dated_reminders/dated_reminders_add.php

```diff
@@ -306,138 +306,196 @@ function selectAll(){
+                                        $optionTxt = '';
+                                        foreach ($dateRanges as $val => $txt) {
+                                            $optionTxt .= '<option value="' . attr($val) . '">' . text($txt) . '</option>';
+                                        }
+                                        echo $optionTxt;
+                                        ?>
+                                    </select>
+                                </div>
+                            </div>
+
+                            <div class="form-group mt-3">
+                                <label for="priority"><?php echo xlt('Priority') ?>:</label>
+                                <div class="mt-2">
+                                    <div class="custom-control custom-radio custom-control-inline">
+                                        <input <?php echo ($this_message['message_priority'] == 3 ? 'checked="checked"' : '') ?> type="radio" class="custom-control-input" name="priority" id="priority_3" value='3'>
+                                        <label class="custom-control-label" for="priority_3"><strong><?php echo xlt('Low{{Priority}}') ?></strong></label>
+                                    </div>
+                                    <div class="custom-control custom-radio custom-control-inline">
+                                        <input <?php echo ($this_message['message_priority'] == 2 ? 'checked="checked"' : '') ?> type="radio" class="custom-control-input" name="priority" id="priority_2" value='2'>
+                                        <label class="custom-control-label" for="priority_2"><strong><?php echo xlt('Medium{{Priority}}') ?></strong></label>
+                                    </div>
+                                    <div class="custom-control custom-radio custom-control-inline">
+                                        <input <?php echo ($this_message['message_priority'] == 1 ? 'checked="checked"' : '') ?> type="radio" class="custom-control-input" name="priority" id="priority_1" value='1'>
+                                        <label class="custom-control-label" for="priority_1"><strong><?php echo xlt('High{{Priority}}') ?></strong></label>
+                                    </div>
+                                </div>
+                            </div>
+
+                            <div class="section-header mt-4 mb-2" style="max-width: 100%; box-sizing: border-box; overflow: hidden;">
+                                <h6 class="text-muted"><?php echo xlt('Message Content');?></h6>
+                            </div>
+
+                            <div class="form-group">
+                                <label for="message"><?php echo xlt('Type Your Message Here');?>:</label>
+                                <textarea onKeyDown="limitText(this.form.message,this.form.countdown,<?php echo attr(addslashes($max_reminder_words)); ?>);"
```

Comment:
super odd escaping here (not your doing since was old code). rec removing attr(addslashes()) and substituting with attr_js()